### PR TITLE
Nataliia/fix styles

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**

### CODE REVIEW

File: source/theme/layout.html [hunk 1]
```diff
@@ -34,13 +34,15 @@
   {% include 'navigation.html' %}
 
   <div class="l-wrapper l-wrapper--fluid main">
+    <div class="l-wrapper__left-padding"></div>
     <div class="main__inner" data-enhancer="sidebar">
-      <nav class="main__sidebar">
-        <div class="sidebar js-mobile-sidebar">
+      <nav class="main__sidebar" id="main-sidebar">
```
Comment (paszkowskiDamian) [inline source/theme/layout.html]: Why did you add `id`? I cannot find any use of that.

File: source/theme/styles/components/_code-example.scss [hunk 2]
```diff
@@ -48,14 +48,49 @@
 }
 
 .rst-content .highlighttable {
-  border: 1px solid $grey-medium;
+  background: #F2F3F4;
```
Comment (paszkowskiDamian) [inline source/theme/styles/components/_code-example.scss]: We are using `scss` please use variables for colors.

File: source/theme/layout.html (hunk 1 above)
Comment (NataliiaKononikhina) [reply source/theme/layout.html]: There is function` toggleBurgerMenu` in `navigation.html`, where I use this id.

File: source/theme/styles/components/_code-example.scss (hunk 2 above)
Comment (paszkowskiDamian) [reply source/theme/styles/components/_code-example.scss]: I can see this color repeating at least twice.

**Comment by jagodapp** [comment]
All of the `svg`s are missing. You have to add them as well.
